Metadata/DynamicFormMetadataLoader.php

```diff
@@ -87,49 +51,49 @@ public function __construct(
     public function warmUp($cacheDir, ?string $buildDir = null): array
     {
         $resource = __DIR__ . '/../Resources/config/forms/form_details.xml';
-        $formMetadataCollection = $this->formXmlLoader->load($resource);
-        foreach ($formMetadataCollection->getItems() as $locale => $formMetadata) {
-            $section = new SectionMetadata('formFields');
-            $section->setLabel($this->translator->trans('sulu_form.form_fields', [], 'admin', $locale));
-            $fields = new FieldMetadata('fields');
-            $fields->setType('block');
-
-            $types = $this->formFieldTypePool->all();
-
-            $fieldTypeMetaDataCollection = [];
-            foreach ($types as $typeKey => $type) {
-                $fieldTypeMetaDataCollection[] = $this->loadFieldTypeMetadata($typeKey, $type, $locale);
-            }
-
-            \usort($fieldTypeMetaDataCollection, static function(FormMetadata $a, FormMetadata $b): int {
-                return \strcmp($a->getTitle(), $b->getTitle());
-            });
-
-            foreach ($fieldTypeMetaDataCollection as $fieldTypeMetaData) {
-                $fields->addType($fieldTypeMetaData);
-            }
-
-            $fields->setDefaultType(\current($fields->getTypes())->getName());
-            $section->addItem($fields);
-            $formItems = $formMetadata->getItems();
-            $this->arrayInsertAtPosition($formItems, 1, [$section->getName() => $section]);
-            $formMetadata->setItems($formItems);
-            $configCache = $this->getConfigCache($formMetadata->getKey(), $locale);
-            $configCache->write(\serialize($formMetadata), [new FileResource($resource)]);
+        $formMetadata = $this->formXmlLoader->load($resource);
+        $section = new SectionMetadata('formFields');
+        foreach ($this->locales as $locale) {
+            $section->setLabel($this->translator->trans('sulu_form.form_fields', [], 'admin', $locale), $locale);
         }
+        $fields = new FieldMetadata('fields');
+        $fields->setType('block');
+
+        $types = $this->formFieldTypePool->all();
+
+        $fieldTypeMetaDataCollection = [];
+        foreach ($types as $typeKey => $type) {
+            $a = $this->loadFieldTypeMetadata($typeKey, $type);
+            $fieldTypeMetaDataCollection[$a->getTitle('en')] = $a;
+        }
+        Assert::notEmpty($fieldTypeMetaDataCollection, 'No field type metadata loaded');
```

Comment:
We're using `addType` that's why we don't need to generate a key explicitly in the array.

```suggestion
        $fieldTypeMetaDataCollection = [];
        foreach ($types as $typeKey => $type) {
            $fieldTypeMetaDataCollection[] = $this->loadFieldTypeMetadata($typeKey, $type);
        }
        Assert::notEmpty($fieldTypeMetaDataCollection, 'No field type metadata loaded');
```